tests/Microsoft.Identity.Test.Common/Core/Helpers/CoreAssert.cs

```diff
@@ -47,6 +49,68 @@ public static void AreEqual(DateTimeOffset expected, DateTimeOffset actual, Time
             Assert.IsTrue(DictionariesAreEqual(dict1, dict2, valueComparer));
         }
 
+        public static void InterfaceExposesObject(Type theObj, Type theInterface, string[] except = null)
```

Comment:
Good idea :)
Of course sometimes a class implements 2 interfaces (See `TokenAcquisition` implements `ITokenAcquisition` and `ITokenAcquisitionInternal` in Microsoft.Identity.Web), but the private one can be implemented explicitly, and therefore the members don't need to be public.

Reply:
Exactly, explicit interface implementations are different. `TokenCache` also implements `ITokenCache` and `ITokenCacheInternal`

Reply:
So maybe an extension method?
Otherwise shall we bump-up the version to 5.0 ?